File: app/src/main/java/com/pdamkotasmg/happywork/fitur/splash/model/packageName/PackageNameRootModel.java
package com.pdamkotasmg.happywork.fitur.splash.model.packageName;

import com.google.gson.annotations.SerializedName;

public class PackageNameRootModel{

    @SerializedName("data")
    private Data data;

    @SerializedName("message")
    private String message;

    @SerializedName("status")
    private Integer status;

    public void setData(Data data){
        this.data = data;
    }

    public Data getData(){
        return data;
    }

    public void setMessage(String message){
        this.message = message;
    }

    public String getMessage(){
        return message;
    }

    public void setStatus(Integer status){
        this.status = status;
    }

    public Integer getStatus(){
        return status;
    }
}

File: app/src/main/java/com/pdamkotasmg/happywork/fitur/splash/model/packageName/PackageNameRootModel.kt
package com.pdamkotasmg.happywork.fitur.splash.model.packageName

import com.google.gson.annotations.SerializedName

data class PackageNameRootModel(

    @field:SerializedName("data")
    val data: Data? = null,

    @field:SerializedName("message")
    val message: String? = null,

    @field:SerializedName("status")
    val status: Int? = null
)